dwh/validate_indicators.sh

```diff
@@ -79,5 +80,21 @@ if [[ "${PVLS_ratio}" != "0.723,0.277,0,0.0851" ]]; then
   exit 1
 fi
 
+# TX_NEW Test
```

Comment:
Now that we have 3 sections like this, let's do the TODO on line 69, i.e., refactor these 3 awk sections into a function.